inc/Alien/Proj4/Makefile.PL

```diff
@@ -1,99 +0,0 @@
-# Proj4 detection process (need to rework using Devel::CheckLib)
-# 
-# (0) check WITH_PROJ
```

Comment:
Not that one line in particular! :-P

Sorry if I wasn't clear. There seem to be some ideas for future improvements in these comments such as searching for `proj4.dll` and making sure that the build can handle both static and dynamic libs.